cli/src/main.rs

```diff
@@ -55,8 +54,245 @@ pub fn main() {
 
     let regtest_mode_enabled = matches.is_present("regtest");
     if regtest_mode_enabled {
-        regtest::launch()
-        // do the regtest
+        use std::ffi::OsString;
```

Comment:
Is this a git rebase reversion?

Reply:
You caught me...  but it's already fixed.